sql/core/src/main/scala/org/apache/spark/sql/api/python/PythonSQLUtils.scala

```diff
@@ -34,17 +34,36 @@ private[sql] object PythonSQLUtils {
   }
 
   /**
-   * Python Callable function to convert ArrowPayloads into a [[DataFrame]].
+   * Python callable function to convert an RDD of serialized ArrowRecordBatches into
+   * a [[DataFrame]].
    *
-   * @param payloadRDD A JavaRDD of ArrowPayloads.
-   * @param schemaString JSON Formatted Schema for ArrowPayloads.
+   * @param arrowBatchRDD A JavaRDD of serialized ArrowRecordBatches.
+   * @param schemaString JSON Formatted Spark schema for Arrow batches.
    * @param sqlContext The active [[SQLContext]].
    * @return The converted [[DataFrame]].
    */
-  def arrowPayloadToDataFrame(
-      payloadRDD: JavaRDD[Array[Byte]],
+  def arrowStreamToDataFrame(
```

Comment:
This seems being a private method now?

Reply:
it's public so it can be called in Python with Py4j

Reply:
oh right, this is only called by the function below so I suppose we don't even need it..